### DISCUSSION TITLE
What is your favorite Japanese movie? 好きな日本の映画はなんですか？

### DISCUSSION THREAD

**Discussion by YuseiNaito** [NAIST-SE/AttractingContributors#425] [General]
Add your favorite Japanese movies to this repository!
https://github.com/YuseiNaito/AttractingContributors-Japanese-Movie

**Comment by aru1702** [comment]
My favorite are Spirited Away (千と千尋の神隠し) and Doraemon: Nobita's Treasure Island (映画ドラえもん のび太の宝島)
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
